spec/views/application/_sidebar.html.erb_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe 'application/_sidebar.html.erb', type: :view do
+  before do
+    assign(:organisations, [])
+    assign(:taxonomies, [])
+  end
+
+  context "filtering organisations" do
+    it "will render select boxes with options for 2 organisations and 1 empty if supplied with list of 2 organisations" do
+      organisations = [build(:organisation), build(:organisation)]
+      assign(:organisations, organisations)
+      render
+
+      expect(rendered).to have_selector('form select#organisation_id option', count: 3)
```

Comment:
What about checking here that we are using the organisation title to render the combo?